LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -221,16 +220,15 @@ window.test_driver_internal.send_keys = function(element, keys)
 
     for (const key of keyList)
         eventSender.keyDown(key, modifiers);
-    return Promise.resolve();
 }
 
-window.test_driver_internal.click = function (element, coords)
+window.test_driver_internal.click = async function (element, coords)
 {
     if (!window.testRunner)
-        return Promise.resolve();
+        return;
 
     if (!window.eventSender)
-        return Promise.reject(new Error("window.eventSender is undefined."));
+        throw new Error("window.eventSender is undefined.");
 
     if (testRunner.isIOSFamily && testRunner.isWebKit2) {
         return new Promise((resolve) => {
```

Comment:
should be `await new Promise((resolve) => {...});` followed by `return;` to preserve old behavior.